WORKSPACE

```diff
@@ -297,3 +297,12 @@ http_archive(
     strip_prefix = "clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04",
     url = "http://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz",
 )
+
+# Roughtime
+new_git_repository(
```

Comment:
What do we need from `new_git_repository` that `git_repository` does not support? I thought `new` was added during a transition period, but now it's fine to use the non-new version?

Reply:
The build_file attribute is not supported on git_repository and I need a custom build file to change the visibility of the roughtime targets.